data/wanderers middle.txt

```diff
@@ -1768,6 +1768,85 @@ mission "Wanderers: Sestor Search: Varu K'prai"
+			choice
+				`	"You're lucky as hell to be here today."`
+					goto glad
+				`	"Thanks for the information. I think that these hostile ships are the ones I'm hunting."`
+					goto thanks
+			
+			label glad1
+			`	"I am! The attacking ships weren't big, but they were tough: our four heavy laser turrets couldn't even produce a dent in their shields," the captain says. "We even hired a beefy Leviathan to protect us from the pirate raids as we travelled to Farpoint, but when these hostile ships jumped in from the northwest, they made short work of our escort and nearly disabled our ship, too. I had to jump out of the system while in atmosphere to save the ship, but now I'm in debt to all my crew because I didn't meet a rush delivery deadline."`
+			choice
+				`	"You really are lucky as hell to be here today."`
+					goto glad
+				`	"Thanks for the information. I think that these hostile ships are the ones I'm looking for."`
+					goto thanks
+			
+			label thanks
+			`	"No problem, kiddo," the captain says, although he looks awed that you are even attempting to fight such powerful ships. "Remember, northwest of Farpoint. If they're anarchists of any sort, they'll be hiding in one of those pirate systems up there. Take care of yourself out there; those are some dangerous ships that jumped us." With that, he bids you farewell.`
+					decline
+					
+			label glad
+			`	"Yeah, those ships would have obliterated us if we hadn't jumped pronto." The pilot looks at you, then asks quizzically, "Say, do you know anything about those ships?"`
+			choice
+				`	"Yeah, those were Korath ships that attacked you."`
+				`	"Maybe. I have a feeling that they're the alien ships that I'm hunting."`
+				`	"I don't want to jump to any conclusions."`
+					goto no
+					
+			`	"Aliens?" The pilot's face reddens, and his eyes narrow in anger. "Like those detestable squirrels, you mean? Told me they were peaceful, but when I went further north, I nearly had my ship torn to pieces!" You try to tell him that the Hai are a separate species from the Korath, but he clearly does not want to hear any more, so you decide to bring the conversation to an end.`
+				goto end
+				
+			label no
+			`	The pilot seems disappointed in your response, so you decide to bring the conversation to an end.`
+				goto end
+				
+			label end
+			`	"Thank you for your time," you say to the captain. The pilot sobers up again, and says, "No problem. Remember, northwest of Farpoint. If they're anarchists of any sort, they'll be hiding in one of those pirate systems up there. Take care of yourself out there; those were dangerous ships we faced." With that, he bids you farewell.`
```

Comment:
I don't see where you did note this before, but you are correct. I'll make the change soon.